pkg/user/repository/bean.go

```diff
@@ -24,3 +24,7 @@ const (
 	WorkflowPValUpdateKey     PValUpdateKey = "Workflow"
 	WorkflowObjPValUpdateKey  PValUpdateKey = "WorkflowObj"
 )
+
+const (
+	EMPTY = ""
```

Comment:
empty what. empty is subjective term....values can be same but definition of empty can be different for different usecases

Reply:
also where is this used, can not find usages

Reply:
Renamed it accordingly. Used as empty placeholder in orm query arguments at multiple places